### PR TITLE
Update to Python 3.10.0

### PR DESCRIPTION
**Pull request by rco-ableton**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -162,10 +160,6 @@ rich==10.12.0
     #   molecule
 ruamel.yaml==0.17.16
     # via ansible-lint
-selinux==0.2.1
-    # via
-    #   molecule
-    #   molecule-docker
```
Comment (ala-ableton) [inline requirements-dev.txt:168]: Based on https://github.com/ansible-community/molecule/blob/54518f9bb1e70e527eee135be9d96b29a88f37c2/setup.cfg#L88-L89 and https://github.com/ansible-community/molecule-docker/blob/acb7d4ef0ea83d822f01adf26af543fa0817d5bd/setup.cfg#L67-L68, `selinux` is only necessary on Linux, but you ran `pip-compile` on macOS...

Comment (rco-ableton) [reply requirements-dev.txt]: Good catch—see https://github.com/Ableton/ansible-role-amtool/pull/35/commits/c3064fa544fdab6f4384a3a162398f71cd70130c, which is generated on a Linux machine (and actually results in no changes to any dependencies).

**Review by ala-ableton — APPROVED** [review]
Once rebased, LGTM 👍🏻

**Comment by rco-ableton** [comment]
@ablbot autosquash

**Pull request merged**